app/upgrades/v320/upgrade.go

```diff
@@ -0,0 +1,50 @@
+package v310
+
+import (
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	"github.com/desmos-labs/desmos/v3/app/upgrades"
+)
+
+var (
+	_ upgrades.Upgrade = &Upgrade{}
+)
+
+// Upgrade represents the v3.2.0 upgrade
+type Upgrade struct {
+	mm           *module.Manager
+	configurator module.Configurator
+}
+
+// NewUpgrade returns a new Upgrade instance
+func NewUpgrade(mm *module.Manager, configurator module.Configurator) *Upgrade {
+	return &Upgrade{
+		mm:           mm,
+		configurator: configurator,
+	}
+}
+
+// Name implements upgrades.Upgrade
+func (u *Upgrade) Name() string {
+	return "v3.2.0"
+}
+
+// Handler implements upgrades.Upgrade
+func (u *Upgrade) Handler() upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		// Nothing to do here as this update does not include anything particular
+		return u.mm.RunMigrations(ctx, u.configurator, fromVM)
+	}
+}
+
+// StoreUpgrades implements upgrades.Upgrade
+func (u *Upgrade) StoreUpgrades() *storetypes.StoreUpgrades {
+	return &storetypes.StoreUpgrades{
+		Added: []string{
```

Comment:
Shouldn't it be `Deleted`?

Reply:
You're totally right! Thanks for spotting this :pray: My bad